lib/docker/template/cache.rb

```diff
@@ -17,12 +17,10 @@ def context(builder, context)
         builder.repo.cache_dir.rm_rf
         $stderr.puts Simple::Ansi.yellow(format("Copying context for %s", builder.repo))
         cache_dir = builder.repo.cache_dir
-        cache_dir.parent.mkdir_p
+        cache_dir.mkdir_p
```

Comment:
Then there should be an issue created with more information on what you are doing with logs so that we can track down why the cache is not being created.  If a build fails we do not ultimately cache, and we also don't cache some types of repositories.  Though this change does little to fix that problem.